kview/src/main/java/dev/ikm/komet/kview/events/CreateStampEvent.java

```diff
@@ -0,0 +1,10 @@
+package dev.ikm.komet.kview.events;
+
+import dev.ikm.tinkar.events.Evt;
+import dev.ikm.tinkar.events.EvtType;
+
+public class CreateStampEvent extends Evt {
```

Comment:
Can you create one event Called `StampEvent` and have event types?
`CREATE_STAMP`, `ADD_STAMP`, etc.